includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -23,6 +23,27 @@ class Sensei_Analysis_Overview_List_Table extends Sensei_List_Table {
 	 */
 	private $post_type = 'course';
 
+	/**
+	 * The average grade for all the lessons in all the courses.
+	 *
+	 * @var int
+	 */
+	private $total_average_grade = 0;
```

Comment:
I took a similar but slightly different approach [here](https://github.com/Automattic/sensei/pull/4904/commits/71e357afe814b36bc0d985f156c2436cd9a37a58). I like the grade property in the `Grading` class better since that seems like a better place for it to live. What do you think about it?

Reply:
Hey Donna, I will remove it altogether since your's PR will be fixing this issue so I will not store/cache data anywhere. So once we have merged both PRs it's gonna be fine.